db/migrate/20190815153504_create_carriages.rb

```diff
@@ -1,10 +0,0 @@
-class CreateCarriages < ActiveRecord::Migration[5.2]
```

Comment:
It comments doesn't need fixe but It very bad practices delete migration please don't do it in future.